scipy/fft/_pocketfft/pypocketfft.cxx

```diff
@@ -28,12 +28,12 @@ namespace py = pybind11;
 using ldbl_t = typename std::conditional<
   sizeof(long double)==sizeof(double), double, long double>::type;
 
-auto c64 = py::dtype("complex64");
-auto c128 = py::dtype("complex128");
-auto clong = py::dtype("longcomplex");
-auto f32 = py::dtype("float32");
-auto f64 = py::dtype("float64");
-auto flong = py::dtype("longfloat");
+using c64 = std::complex<float>;
```

Comment:
I don't think I understand the relative merits of `using c64 = std::complex<float>;` vs `auto c64 = py::dtype("complex64");` .  Was that just a stylistic choice or was that change necessary for `py::isinstance` to work?

Reply:
`using` declares a type alias, similar to `typedef`.